vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -151,6 +151,7 @@
       <AdditionalOptions>%(AdditionalOptions) /noattributename</AdditionalOptions>
     </Midl>
   </ItemDefinitionGroup>
+  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\PkgESCompliance.props" />
```

Comment:
This doesn't address the issue raised by #9259 on line 146.

Reply:
Have a separate PR that cleans this up with HermesNoLink.

I gave some feedback in the issue about build logic customization.

The specific example might be the kind of thing we should enable for the whole platform, depending on its affect to build performance and symbol output size.